spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -1384,14 +1384,29 @@ def find_connection_file(self, connection_file):
 
     # ---- Public API
     # -------------------------------------------------------------------------
```

Comment:
Please move this block comment to be above `find_connection_file` because that's also public API for this widget.